packages/integration-tests/tests/bridge_mainnet_tokens.ts

```diff
@@ -160,6 +161,9 @@ describe('Bridge mainnet tokens', function () {
   }
 
   before(async () => {
+    // spinning up all docker containers
```

Comment:
I feel this is quite repetitive. we can wrap the tests within a shared function and use it. 

something like this:

```
export function testWithContainers(name: string, fn: () => void): void {
  describe(name, function() {
    before(async => () {
      await startContainers();
    });
    
    fn();
     
   after(async () => {
      await stopComtainers();
   )
  });
```

in the tests we can replace `describe()` with `testWithContainers()`

```
testWithContainers("bridge_erc20", function () {...}); 
```